doc/user-guide/user-guide.rst

```diff
@@ -0,0 +1,39 @@
+.. _User_Guide:
+
+#######################
+Electrometer User Guide
+#######################
+
+The electrometer is a Keithley Spectrograph that measures the amount of electrons in proximity with a sensor.
```

Comment:
This CSC is used to control Keithley 6417b electrometers. The primary use-case is to read/monitor the electrical signals from a Hamamatsu S2281 photodiode, however it can be used with other devices.
The device is generally used in a current and charge measuring modes.